CODING_STANDARD

```diff
@@ -171,6 +171,23 @@ Output:
   (status(), error(), debug(), etc)
 - Use '\n' instead of std::endl
 
+Unit tests:
+ - Unit tests are written using Catch: https://github.com/philsquared/Catch/
+ - For large classes:
+    - Create a separate file that contains the tests for each method of each
+      class
+    - The file should be put inside unit/class/path/class_name/function_name.cpp
```

Comment:
The file should be named according to unit/class/path/class_name/function_name.cpp